### ISSUE TITLE
frac divider not rendering however above works?

### ISSUE THREAD

**Issue by mymonoo** [lingarajsankaravelu/Katex#38]
Hi. This was working perfect. But since last few days the \frac does not rendering the divider but \above 1pt works. Any suggestion how it got messed up? I have removed all my css but still. I tried searching but no one seems to have such issue before. Its working perfect in the katex website tho. I have tried removing all my css files from the document but still having same issue.
This is the math I am trying to render 
`\begin{aligned}&=\frac{1}{2\cosec^260\degree(cos^260\degree-3\cos60\degree+2)}\\&=\frac{1}{2\times\frac43\times(\frac14-\frac32+2)}\\&=\frac{1}{2\times\frac43\times(\frac14+\frac12)}\\&={1 \above 1pt 2}\end{aligned}`
![Screenshot_2020-05-03-12-08-21](https://user-images.githubusercontent.com/37544677/80907883-2c59f780-8d38-11ea-83fb-5c99574c5c15.png)

**Comment by ghost** [comment]
Hi, did you find any solution? I have the same problem.

**Comment by lingarajsankaravelu** [comment]
> Hi, did you find any solution? I have the same problem.

 No. But feel free to pick this issue up and create a pull request once you were able to find a solution from your end.

**Comment by ghost** [comment]
Surprisingly, it works for me now: $ \\\frac{1}{2} $.

Edit: but not always :/

Edit: I know! Text size have to be big enough (about 16sp).

**Comment by Junaid2005** [comment]
Why is it double \\

> Surprisingly, it works for me now: $ \\frac{1}{2} $.
